publication/ver11/cr/security_and_privacy.md

```diff
@@ -0,0 +1,785 @@
+reveal private security data to WoT Applications or to other devices.
+
+For selectively accessible sensitive information,
+Discovery defines a web service to provide TDs and some Things described by 
+TDs may need to manage sensitive data internally and provide a network interface 
+equivalent to a web service that provides selective access to it.
+Both cases should manage sensitive information using known best practices i
+for web services, including encrypting data at rest, using secure transport, and enforcing access control.
+
+As noted above, in some cases TDs themselves may be considered "sensitive information",
+for example if they describe known-vulnerable devices.  
+This may be the case during development, for example.
+In this case appropriate access controls should be put in place to restrict
+access to TDs.  This is supported by the WoT Discovery specification, just like 
+any other web service managing access to sensitive information. 
+Known-vulnerable Things
+themselves should be protected by additional means such as a segmented network,
+an approach also described in the Architecture document.
+
+These issues are discussed in the Security Considerations sections of the 
+various specifications.
+
+### 5 [Do the features in your specification introduce new state for an origin that persists across browsing sessions?](https://www.w3.org/TR/security-privacy-questionnaire/#persistent-origin-specific-state)
+This is not a browser-oriented specification although
+WoT Things can be accessed through browsers as if they were web services. 
+However, no new state is introduced or required in browsers beyond
+current mechanisms (cookies, OAuth sessions, etc).
+A WoT Thing can be thought of as simply another web service, so no *new* mechanisms are defined.
+An IoT device can also be a client of a WoT Thing but in this case
+implements only the functionality required by HTTP and the interface described in
+the TD. 
+Clients only support existing mechanisms (such as OAuth sessions) already supported
+by browsers. 
+No "new" mechanisms are defined *by the specification* to retain state.
+
+A given client may, however, choose to retain state between sessions
+since in IoT devices there is no clear definition of "session".
+In fact, such data retention
+may be part of the device's purpose (for example, a device designed to log history data
+from a sensor).
+
+We also define a Directory web service as part of Discovery whose
+*purpose* is to retain information about other Things and make it available to 
+other Things with suitable access rights.
+This web service however includes an API that allows the client to manage what state is
+retained and to delete it if necessary.
+
+### 6 [Do the features in your specification expose information about the underlying platform to origins?](https://www.w3.org/TR/security-privacy-questionnaire/#underlying-platform-data)
+If we map "platform (browser)" to WoT Consumer and "origin" to WoT Thing then the
```

Comment:
well, exactly.  Which is why I give my interpretation of what they "mean" in the answer.  By platform here they mean the computer/OS the browser (client) is running on.   If we take that literally we can map "client" to consumer, but what they REALLY want to know is whether a company can track people by fingerprinting details of the platform, OR if an attacker (from a malicious web server) can find out information about a browser/platform (for example, use of an older version of something) to craft an attack.  So I have to interpret what they actually "mean" (the linked discussion is helpful) and then answer the "actual" concern.

Although maybe my use of "map" here is confusing, perhaps it would be clearer if I said "interpret".

Reply:
reworded to be more explicit, and to interpret "platform" as "the underlying hardware/software of an implementation"